app/uploaders/validate_pdf.rb

```diff
@@ -11,10 +13,9 @@ def validate_pdf(file)
     temp_file = file.tempfile
     return unless temp_file.readpartial(4) == '%PDF'
 
-    PDF::Reader.new(temp_file).info
-  rescue PDF::Reader::MalformedPDFError
-    raise CarrierWave::UploadError, 'PDF is missing an end of file marker'
-  rescue PDF::Reader::EncryptedPDFError
```

Comment:
Is there a reason why this wasnt working to begin with?

Reply:
:heavy_plus_sign: to understanding the original failure w/ `PDF` library

Also if we're replacing the dependency w/ `origami`, can the original library be removed/replaced entirely?

Reply:
So the cause of why it wouldn't trigger the error is still unclear to me. Testing locally, if you run `PDF::Reader.new("encrypted.pdf").info` it would trigger successfully within `irb` - so something about the upload process is causing it to not trigger. As demonstrated in my screenshots though, my fix does indeed catch the PDFs that are encrypted or malformed.

Reply:
@kfrz just doing some checking within the codebase, `PDF::Reader` can't be removed at this time, it's used in some other spots.

Reply:
Hm, I think we should look more into why PDF::Reader is catching encrypted files locally but not when put into actual practice... this is a little disconcerting. Especially if it means we wouldnt need to add, yet, another gem to do work that a pre-existing gem should already be handling. However, I wont let that keep this from being merged in.